src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,44 @@
+.watch {
+  @include circle($size);
+  background-color: bisque;
+  display: flex;
+  position: relative;
+  border: 2px dashed mediumblue;
+  
+  &--content{
+    margin: 100px auto;
+  }
+ 
+  &::after {
+    position: absolute;
+    display: block;
+    content: "";
+    width: 20px;
+    height: 20px;
+    border-radius: 50%;
+    background-color: brown;
+    top: 48%;
+    left: 48%;
+  }
+  
+  &__minute,
+  &__second {
+    position: absolute;
+    display: block;
+    border-radius: 30%;
+    width: 6px;
+    height: $size/2;
+    background-color: burlywood;
+    left: 50%;
+    transform-origin: bottom;
+  }
+  
+  &__second {
+    transform: rotate(90deg);
```

Comment:
you have animation, which starts rotate on 0deg, so you don't need this line

Reply:
Fix your mate's comment